### PR TITLE
WIP: Peer review for #19

### PR DESCRIPTION
**Pull request by tradej**
Small changes I deem appropriate before #19 is merged.

### CODE REVIEW

File: modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc [hunk 1]
```diff
@@ -1,18 +1,19 @@
 [#anchor-and-file-names]
 = Anchor Names and File Names
 
-To optimize modular documentation, follow these guidelines for naming module anchors and files.
+To optimize modular documentation, follow these guidelines for naming module anchors and files:
 
 Anchor names:: Provide an anchor for every module so that it can be identified when reused or cross-referenced. Give the anchor name the same or a similar name to the module heading, separated by underscores, and indicate the module type (`con`, `ref`, `proc`) at the end of the name. If applicable, consider also providing a module sub-type (`create`, `edit`, `delete`) before the module type:
 +
+--
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc]: Using an open block to resolve wrong indentation.

File: modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc [hunk 2]
```diff
@@ -1,14 +1,16 @@
 [#reusing-modules]
 = Reusing Modules in Assemblies
 
-When you create content in modules, you can use the same module multiple times in an assembly without having to replicate information. However, in order to facilitate module reuse, you must embed a document attribute variable in the anchor name for the module and then define that attribute in the assembly each time the reused module appears. If the attribute is not embedded and assigned, an error appears at build time reporting the duplicated anchor ID.
+When you create content in modules, you can use the same module multiple times in an assembly without having to replicate information in multiple source files. However, in order to facilitate module reuse, embed a document attribute variable in the anchor name for the module and define that attribute in the assembly before each inclusion of the reused module. If the attribute is not embedded and assigned, an error appears at build time reporting the duplicated anchor ID:
 
 .Error at Build Time When Anchor Has No Attribute
+====
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc]: Making this an _example._

File: modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc [hunk 3]
```diff
@@ -34,9 +36,9 @@ NOTE: Although you can use any document attribute that clearly indicates the var
 = Module B Heading
 ----
 
- . In the assembly file, define the `+++:context:+++` attribute immediately above any included modules that are being reused. How you define the attribute depends on whether the module is included once in multiple assemblies, or is included multiple times in a single assembly.
+ . In the assembly file or the master book file, define the `+++:context:+++` attribute immediately above any included modules that are being reused. How you define the attribute depends on whether the module is included once in multiple assemblies, or is included multiple times in a single assembly.
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc]: This is IMO important. If `{context}` is not set in the `master.adoc` file, we will get weird anchor IDs of modules included until the first assembly that reuses modules appears.

File: modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc [hunk 4]
```diff
@@ -116,16 +120,20 @@ For details, see xref:module_A_being_reused_assembly-1-name[].
...
 The module files contain the following content:
 
 .projects-con.adoc
 [source]
 ----
-// Module included in the following assemblies: "assembly_projects.adoc", "assembly_assets.adoc"
+// Module included in the following assemblies:
+//
+// assembly_projects.adoc
+// assembly_assets.adoc
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc]: Scales better than an inline list.

File: modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc [hunk 5]
```diff
@@ -178,21 +189,24 @@ include::assets-packages-gloss.adoc
...
 
 or
 
+[source]
+----
 For details, see xref:projects_con_assembly-projects[].
-
 ----
 
+[discrete]
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc]: While I can see why we would want the practical examples numbered and shown in the ToC, I believe being able to go to that example without necessarily having read (scrolled) through the intro of the module is undesirable. However, feel free to disregard this suggestion.

File: modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc [hunk 6]
```diff
@@ -221,17 +235,21 @@ include::projects-con.adoc
...
+----
 
 or
 
+[source]
+----
 For details, see xref:projects_con_definitions[].
 ----
 
+[discrete]
```
Comment (tradej) [inline modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc]: IMO, this should definitely not be numbered and/or shown in ToC.

**Comment by sterobin** [comment]
I'm fine with the suggested changes. Okay to merge, IMO. Thanks @tradej.

**Comment by sterobin** [comment]
I've reviewed the diffs and am fine with the changes. This PR can be merged with PR #19.

**Pull request merged**